polaris-core/src/main/java/org/apache/polaris/core/storage/PolarisStorageIntegrationProviderImpl.java

```diff
@@ -16,29 +16,26 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.polaris.service.storage;
+package org.apache.polaris.core.storage;
```

Comment:
Why move an implementation class to core?

Reply:
This was leftover from a previous attempt. I changed it back.